src/plugins/finalcutpro/inspector/video.lua

```diff
@@ -160,16 +159,19 @@ local function doStabilizationMethod(value)
             return true
         end
     end):Is(true):Then(
-        If(stabilization:doShow()):Is(true)
+        If(stabilization:doShow())
         :Then(
-            If(function() return stabilization:isShowing() end):Is(true)
+            If(stabilization.isShowing)
             :Then(
-                If(function() return stabilization.enabled:checked() end):Is(false)
+                If(stabilization.enabled.checked):Is(false)
                 :Then(stabilization.enabled:doCheck())
-                :Then(WaitUntil(stabilization):Is(true):TimeoutAfter(2000))
+                :Then(WaitUntil(stabilization.enabled):Is(true):TimeoutAfter(2000))
+            )
+            :Then(
+                If(function() return method():UI():enabled() end) -- Only try and "tick" it if it's enabled. The stabilisation might still be processing.
```

Comment:
The `PopUpButton` subclasses `Element` and inherits its `isEnabled` `cp.prop`, so we can reference that directly:

Reply:
```suggestion
                If(method.isEnabled)) -- Only try and "tick" it if it's enabled. The stabilisation might still be processing.
```